### ISSUE TITLE
Display players waiting on match still in the tournament

### ISSUE THREAD

**Issue by emtichenor** [emtichenor/cube-tournament#11]
At the beginning of each round we are currently printing the matches for that round. We would also like to see the players that are still in the tournament and waiting on a match.

Ideally they would be grouped by what round they are in or at least if they are in winners or losers, but we may not have an easy way to know this. If we dont we may consider adding a person.current_round but it may be hard to know what that is if they go from winners to losers and are waiting on a match.

**Comment by emtichenor** [comment]
Few bugs:
Awkward in handling uneven tournament sizes (not power of 2)
If theres no winners matches it displays "Winners Matches: None", I want to just not print anything
Might be a bug with DT class, seems with awkward number of people sent to losers bracket the lowest seed gets an advantage when it should be the highest. (Seen at size=13)

**Comment by emtichenor** [comment]
New issue with printing people in the same round in later rounds
